src/UI/Panels/ClipboardPanel.cs

```diff
@@ -19,25 +20,76 @@ public class ClipboardPanel : UEPanel
         public override bool ShouldSaveActiveState => true;
         public override bool ShowByDefault => true;
 
-        private static Text CurrentPasteLabel;
+        private static List<GameObject> clipboardRows = new List<GameObject>();
 
         public ClipboardPanel(UIBase owner) : base(owner)
         {
         }
 
+        public static void SelectClipboardItem(int index)
+        {
+            selectedItem = index;
+        }
+
+        /// <summary>
+        /// Adds a new entry and selects it
+        /// </summary>
+        public static void AddNewEntry()
```

Comment:
Is this a test function? If so, please remove it.

Reply:
This adds a new entry to the end of clipboard and selects it. It's mostly used by the UI, but is necessary if you want to stay away from using the Console for everything.